src/sentry/api/paginator.py

```diff
@@ -493,13 +493,21 @@ def __init__(self, queryset, order_by):
         try:
             instance = queryset[:1].get()
             self.instance_type = type(instance)
-            assert hasattr(
-                instance, self.order_by
-            ), f"Model of type {self.instance_type} does not have field {self.order_by}"
-            self.order_by_type = type(getattr(instance, self.order_by))
+            if type(self.order_by) == list:
+                for key in self.order_by:
+                    self._assert_has_field(instance, key)
+                self.order_by_type = list
+            else:
+                self._assert_has_field(instance, self.order_by)
+                self.order_by_type = type(getattr(instance, self.order_by))
```

Comment:
It might be better to just always convert `order_by` to a list so that we don't need this branching logic throughout this code.

Reply:
Yeah you're right...I feel like I did this to retain some of the order_by_type logic but I changed that instead (order_by_type is a list now unless order_by is a list of length 1, then its the type of whatever that field is)